$OpenBSD: patch-gen_pattern_cc,v 1.1 2019/03/17 15:01:21 cwen Exp $
ports-gcc fix for:
gen_pattern.cc:622:28: error: 'strcmp' was not declared in this scope
Index: gen_pattern.cc
--- gen_pattern.cc.orig
+++ gen_pattern.cc
@@ -26,6 +26,7 @@
 #include <math.h>
 #include <unistd.h>
 #include <fcntl.h>
+#include <string.h>
 #include <sys/stat.h>
 #include <sys/types.h>
 
